### PR TITLE
feat(notifier): edit provider

### PR DESCRIPTION
**Pull request by vjgene**
Add logic to update plans existing plans editing a provider url
Deactivate plan ids that are not present in the new url

### CODE REVIEW

File: src/migrations/scripts/1615470672120-notifier-init.ts [hunk 1]
```diff
@@ -111,6 +111,12 @@ const migrationCommands = function (transaction: any): Commands {
             type: Sequelize.INTEGER,
             field: 'id',
             primaryKey: true,
+            allowNull: false,
+            autoIncrement: true
+          },
+          planId: {
+            type: Sequelize.INTEGER,
+            field: 'planId',
```
Comment (jurajpiar) [inline src/migrations/scripts/1615470672120-notifier-init.ts:119]: The notifier initial migration already exists on staging. We should verify that staging cache db gets deleted upon deployment. If it doesn't create a new migration script for this change.

File: src/services/notifier/handlers/provider.ts [hunk 2]
```diff
@@ -12,15 +12,33 @@ import { NotifierSvcProvider } from '../notifierService/provider'
...
 
     try {
       await providerService.update(provider, { provider, url })
+      try {
+        const currentPlans = await PlanModel.findAll({
+          where: { providerId: provider },
+          include: [{ model: NotifierChannelModel }, { model: PriceModel }]
+        })
+        const [host, port] = url.split(/(?::)(\d*)$/, 2)
+        const svcProvider = new NotifierSvcProvider({ host, port })
```
Comment (jurajpiar) [inline src/services/notifier/handlers/provider.ts]: refactor out

File: src/services/notifier/handlers/provider.ts [hunk 3]
```diff
@@ -12,15 +12,33 @@ import { NotifierSvcProvider } from '../notifierService/provider'
...
+        const { content: incomingPlans } = await svcProvider.getSubscriptionPlans() || {}
```
Comment (jurajpiar) [inline src/services/notifier/handlers/provider.ts]: not sure how this is passing linter check as `{}` does not contain `content` key

**Review by jurajpiar — APPROVED** [review]
looking good.

File: src/services/notifier/handlers/provider.ts [hunk 4]
```diff
@@ -12,15 +12,33 @@ import { NotifierSvcProvider } from '../notifierService/provider'
 import SubscriptionModel from '../models/subscription.model'
 import { getTokenSymbol } from '../../utils'
 import BigNumber from 'bignumber.js'
+import { NotifierProviderError } from '../../../errors'
```
Comment (itofarina) [inline src/services/notifier/handlers/provider.ts]: Please, remove unused import reported by lgmt bot

**Review by itofarina — APPROVED** [review]
LGTM 👌 - Please fix the lgtm bot alert

**Pull request merged**